dist/node.js

```diff
@@ -33,6 +33,7 @@ class node {
   constructor (options = {}) {
     Object.assign(this, {
       network: 'testnet', /** the network to connect to. */
```

Comment:
using the same method as storage to override this attribute (use seeds instead of nodes):
```
network: {
  network: 'privatenet',
  seeds: [
    { domain, port}
  ]
}
```